### PR TITLE
Fix example to use ESM

### PR DESCRIPTION
**Pull request by coderaiser**
### Initial checklist

*   [x] I read the support docs 
*   [x] I read the contributing guide 
*   [x] I agree to follow the code of conduct 
*   [x] I searched issues and couldn’t find anything (or linked relevant results below) 
*   [x] If applicable, I’ve added docs and tests

### Description of changes

Updated examples from `README` to `ESM`

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -24,17 +24,17 @@ npm install unist-util-visit
 ## Use
 
 ```js
-var u = require('unist-builder')
-var visit = require('unist-util-visit')
+import u from 'unist-builder'
+import visit from 'unist-util-visit'
```
Comment (wooorm) [inline readme.md:28]:
```suggestion
import {u} from 'unist-builder'
import {visit} from 'unist-util-visit'
```

^-- they are named exports now

**Review by wooorm — CHANGES_REQUESTED** [review]

**Review by wooorm — APPROVED** [review]

**Comment by wooorm** [comment]
thank you!

**Comment by coderaiser** [comment]
Little bit sad that [v8-compile-cache](https://github.com/zertosh/v8-compile-cache/issues/30) would be unavailable, it doesn't support `ESM`, but used by tools like `postcss`, `eslint`, etc. It boosts about 20% to speed of starting CLI apps.

**Comment by wooorm** [comment]
Yup, and webpack. It seems to be a big bottle neck

**Pull request merged**